cobaya/likelihoods/planck_2018_lowl/TT_native.py

```diff
@@ -0,0 +1,157 @@
+import os
+import numpy as np
+from cobaya.likelihoods.base_classes import InstallableLikelihood
+
+class TT_native(InstallableLikelihood):
+    """
+    Python translation of the Planck 2018 Gibbs TT likelihood (python Eirik Gjerløw, Feb 2023)
+    See https://wiki.cosmos.esa.int/planck-legacy-archive/index.php/CMB_spectrum_%26_Likelihood_Code
+
+    """
+
+    install_options = {"github_repository": "CobayaSampler/planck_native_data",
+                       "github_release": "v1",
+                       "asset": "planck_2018_lowT.zip",
+                       "directory": "planck_2018_lowT_native"}
```

Comment:
Doesn't match zip?

Reply:
Yeah, I didn't really understand this - so the "directory" is meant to be the directory within the zip file once it has been unzipped?

Reply:
Actually maybe that line is OK, more the issue that your zip file has everything in a subdirectory. (I would just compare with e.g. how EE_native and other files are set up.) But if it works already, just ignore me! You can test with "cobaya-install planck_2018_lowl.TT_native"